File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/domain/Orders/Models/OrderTotal.cs
using System;

namespace Registration.Orders
{
    public class OrderTotal
    {
        public OrderLine[] Lines { get; private set; }
        public decimal Total { get; private set; }

        public OrderTotal() { }
        public OrderTotal(OrderLine[] lines, decimal total)
        {
            Lines = lines;
            Total = total;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/orders/Models/OrderTotal.java
package com.microsoft.conference.registration.domain.orders.Models;

import java.math.BigDecimal;
import java.util.List;

public class OrderTotal {
    public List<OrderLine> orderLines;
    public BigDecimal total;

    public OrderTotal() {
    }

    public OrderTotal(List<OrderLine> orderLines, BigDecimal total) {
        this.orderLines = orderLines;
        this.total = total;
    }
}
